loadbalancer/etcd/publisher_test.go

```diff
@@ -0,0 +1,107 @@
+package etcd
```

Comment:
Please use package etcd_test, and import package etcd. (You can disambiguate Go kit's etcd from CoreOS' etcd by import-aliasing the latter as stdetcd, or the former as kitetcd.)